macros/edr/dbt_artifacts/upload_dbt_invocation.sql

```diff
@@ -120,12 +123,50 @@
   {% do return(none) %}
 {% endmacro %}
 
-{% macro get_job_url() %}
+{% macro get_job_url(orchestrator, job_id) %}
   {% set var_value = elementary.get_var("job_url", ["JOB_URL", "DBT_JOB_URL"]) %}
   {% if var_value %}
     {% do return(var_value) %}
   {% endif %}
-  {% if elementary.get_first_env_var(["GITHUB_ACTIONS"]) %}
+  {% if orchestrator == 'airflow' %}
+    {% set server_url = elementary.get_var('orchestrator_domain', ["ORCHESTRATOR_DOMAIN"]) %}
```

Comment:
I think this should be called `AIRFLOW_URL` rather than `ORCHESTRATOR_DOMAIN` because:
1. It's not only the domain, it's the URL, for instance, it needs to include `http://` or `https://`, port, and so on.
2. It only applies to Airflow so we'd want to couple it with the word Airflow.